### PR TITLE
Regenerate with jhipster 7.2.0 and react-mobx blueprint

### PR DESCRIPTION
**Pull request by calvinlu3**
This is related to https://github.com/oncokb/oncokb/issues/2944
- Downgraded to jhipster 7.2, so we can use [generator-jhipster-react-mobx](https://github.com/erkieh/generator-jhipster-react-mobx)
- Generated with skipUserManagement and skipFakeData set to false
- [x] Delete some unnecessary files (ie registration, etc)

### CODE REVIEW

**Comment by calvinlu3** [comment]
Closing because found a way to generate with jhipster 7.3.1

**Comment by calvinlu3** [comment]
@zhx828 When you have the chance could you take a look at the generated files? 

- The issue with `history` that I brought up during standup has to do with the compatibility of `react-router-dom@5 and history@5`.

File: pom.xml [hunk 1]
```diff
@@ -30,9 +30,8 @@
         <!-- Build properties -->
         <maven.version>3.3.9</maven.version>
         <java.version>11</java.version>
-        <!-- we use nvm manages the npm version. npm 6 is the default for node 14 -->
         <node.version>v14.17.6</node.version>
-        <npm.version>6.14.15</npm.version>
+        <npm.version>7.23.0</npm.version>
```
Comment (zhx828) [inline pom.xml:34]: I remember there was an issue using v7 with node 14. Seems no longer the issue?

Comment (calvinlu3) [reply pom.xml:34]: Thanks for pointing that out, I think the npm version change was a result of jhipster generator. Let me double check

**Pull request merged**